src/tests/utils/render.test-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { render, type RenderResult } from "@testing-library/svelte";
+import type { ComponentProps, ComponentType, SvelteComponent } from "svelte";
+
+export const renderWithEvents = <C extends SvelteComponent>(
+  cmp: ComponentType<C>,
+  options: {
+    props?: ComponentProps<C>;
```

Comment:
How about already including `events` in `props` so we offer the same API as the Svelte 5 render function?

Reply:
In Svelte v5 the `events` are not included in the `props` but within the `options` as in this API.

For example.

```
const { container } = render(Checkbox, {
        props,
        events: {
          nnsChange: () => done(),
        },
      });
```

Reply:
Oh, ok, great.
But then what do you mean by the in the other file:
```
// TODO: remove once events is migrated to props
```

Reply:
I can probably rephrase "props" to "callback props" to be more precise.

In Svelte v5 events with dispatcher are deprecated and have to be replaced with callback props. Once we have migrated the events, we can pass callback as props instead of listening to events.

https://svelte.dev/docs/svelte/v5-migration-guide#Event-changes-Component-events

Reply:
So the `events` that we are passing now are at the same time new and already deprecated?

Is that TODO specific to that one test or would it apply to all tests? I don't see what's different about that one test with the TODO.
Anyway, the PR is fine so feel free to merge and my questions can be answered by future PRs.